### PR TITLE
resolve issue #40 aka adding getter on limit and offset

### PR DESCRIPTION
**Pull request by Grummfy**
Here is the #40 PR

### CODE REVIEW

File: src/Common/Select.php [hunk 1]
```diff
@@ -561,6 +561,16 @@ public function unionAll()
         return $this;
     }
 
+	public function getLimit()
+	{
+		return $this->limit;
```
Comment (harikt) [inline src/Common/Select.php]: seems tabs :) .

**Comment by harikt** [comment]
Thank you @Grummfy . You have used 1 tab = 8 character. Please change to 4 space.

Paul will look into this.

**Comment by Grummfy** [comment]
1 tabs = how many characters you want ... it's a configuration of your editor ...
but I change all my tabs to 4 space ... it will be easier if you have a checkstyle file to run phpcsfix...

**Comment by harikt** [comment]
1 tab = 4 charactes .

Thanks!

_Hari K T_

You can ring me : +91 9388 75 8821

http://harikt.com , https://github.com/harikt ,
http://www.linkedin.com/in/harikt , http://www.xing.com/profile/Hari_KT

Skype  : kthari85
Twitter : harikt

**Comment by staabm** [comment]
this is a BC break, should it require a new major version?

**Comment by Grummfy** [comment]
@staabm  nop. jsut add some getters

**Comment by staabm** [comment]
@Grummfy you changed existing interfaces, therefore classes which implement them outside of the lib will get a fatal error after this PR lands

**Comment by harikt** [comment]
@staabm good call :) .

**Comment by pmjones** [comment]
So, is this a BC break or not?

**Comment by harikt** [comment]
@pmjones it seems to be a BC break :-/ . If someone have implemented `LimitInterface.php` and `LimitOffsetInterface.php` they should implement it in those libraries. May be ask this guy https://github.com/tomzx/php-semver-checker

**Comment by pmjones** [comment]
Merged manually, removed the Interface changes that would have been BC breaks.

**Pull request merged**